resources/views/auth/login.blade.php

```diff
@@ -22,48 +22,57 @@
 
 @section('auth_body')
     <form action="{{ $login_url }}" method="post">
-        {{ csrf_field() }}
+        @csrf
 
         {{-- Email field --}}
         <div class="input-group mb-3">
-            <input type="email" name="email" class="form-control {{ $errors->has('email') ? 'is-invalid' : '' }}"
-                   value="{{ old('email') }}" placeholder="{{ __('adminlte::adminlte.email') }}" autofocus>
+            <input type="email" name="email" class="form-control @error('email') is-invalid @enderror"
+                   value="{{ old('email') }}" placeholder="{{ __('adminlte::adminlte.email') }}"
+                   autocomplete="email" autofocus>
```

Comment:
For security reasons, it will be better to disable autocomplete in login forms.

Reply:
done!